source/ceylon/net/http/server/internal/RequestImpl.ceylon

```diff
@@ -159,33 +181,46 @@ class RequestImpl(HttpServerExchange exchange,
     }
 
     variable Map<String, String[]>? lazyQueryParameters = null;
-    value queryParameters => lazyQueryParameters 
+    value queryParameters => lazyQueryParameters
             else (lazyQueryParameters = readQueryParameters());
 
     variable FormData? lazyFormData = null;
-    value formData => lazyFormData 
+    value formData => lazyFormData
             else (lazyFormData = buildFormData()) ;
 
+    shared actual String[]? formParameters(String name) {
+        return formData.parameters[name];
+    }
+
+    shared actual String? formParameter(String name) {
+        if (nonempty x = formData.parameters[name]) {
+            return x.first;
+        }
+        else {
+            return null;
+        }
+    }
+
```

Comment:
We already have `parameters` method, why do you think we should have another method for the same goal?